blscosi/protocol/protocol.go

```diff
@@ -99,13 +99,14 @@ func DefaultThreshold(n int) int {
 }
 
 // DefaultSubLeaders returns the number of sub-leaders, which is the
-// cube-root of the number of nodes.
+// square root of the number of nodes - 1.
 func DefaultSubLeaders(nodes int) int {
-	// As `math.Pow` calculates `8 ** (1/3) < 2`,
+	if nodes == 1 {
+		return 1
+	}
+	// As `math.Pow` calculates `9 ** (1/2) < 3`,
```

Comment:
`math.Pow(9.0, 1.0/2.0) < 3.0 == false` for me

Reply:
Ah, that makes sense in fact. I just copied the lines from the previous `math.Pow(8.0, 1.0/3.0)` which is smaller than `2`. But as you can represent 1/2 in binary, but not 1/3, that's probably the cause. So I remove that.